bookshelf/src/main/webapp/base.jsp

```diff
@@ -34,9 +34,11 @@ Copyright 2015 Google Inc. All Rights Reserved.
         </div>
         <ul class="nav navbar-nav">
           <li><a href="/">Books</a></li>
-          <li><a href="/books/mine">My Books</a></li>
+          <div class="form-group ${isAuthConfigured ? '' : 'hidden'}">
```

Comment:
I did `<c:if test="${isOAuth2Configured}"><li><a href="/books/mine">My Books</a></li></c:if>` in my version (still pending because I'm going through the rest of the tutorial so I can test it. I think `c:if` would be a little more clear than a hidden div.